### ISSUE TITLE
Gelbooru won't download all anymore

### ISSUE THREAD

**Issue by naruhina2016** [Bionus/imgbrd-grabber#3315]
![afbeelding_2024-11-02_235139785](https://github.com/user-attachments/assets/e540e844-2caf-4142-877f-f52aa836cc6d)

I have not been using the grabber for 3 weeks.
Now 3 weeks later it does not work the same anymore...:(
Even danbooru won't download all of it anymore...:(
Please help.
Thanks in Advacne.

---Naruhina---

**Comment by GameDevKitY** [comment]
same issue, have you found a fix?

**Comment by naruhina2016** [comment]
No not yet

**Comment by naruhina2016** [comment]
Weird danbooru never had that problem.

**Comment by GameDevKitY** [comment]
> Hi, first;
> 
> Never go above 20K images in a single task, i'll fail with gelbooru API. use id:< and id:> to select a part of the images you're trying to download.
> 
> And also, you may need your cloudfare cookies, and your username and api keys to use gelbooruAPI.
> 
> And we need to logs to help you, I see they are activated on your screen, so the log file should be into the grabber folder as "main_xxx.log".

yeah thats not the issue, what we were hitting is a gelbooru limit and by doing id:>x it fixes that
